articles/quickstart/spa/vanillajs/_includes/_centralized_login.md

```diff
@@ -179,20 +190,18 @@ Provide a template with controls for the user to log in and log out.
 ```
 
 ::: note
-This example uses Bootstrap styles, but that's unimportant. Use whichever style library you like, or don't use one at all.
+This example uses Bootstrap styles. You can use any style library, or not use one at all.
 :::
 
-The `click` event listeners added to the **Log In** and **Log Out** buttons make the appropriate calls to the functions in `app.js` to allow the user to log in and log out. Notice that these buttons are conditionally hidden and shown depending on whether or not the user is currently authenticated.
-
-When the **Log In** button is clicked, the user will be redirected to Auth0's hosted login page.
+Depending on whether the user is authenticated or not, they see the **Log Out** or **Log In** button. The `click` event listeners on the buttons make calls to functions in the `app.js` file to let the user log in or out. When the user clicks **Log In**, they are redirected to the Auth0 hosted login page. 
 
 <%= include('../../_includes/_hosted_login_customization' }) %>
 
 ## Process the Authentication Result
```

Comment:
In the rest of the guides, there is an 'Add a callback component' section before the 'Process the authentication result'. Shouldn't the JS quickstart contain this section as well?